### ISSUE TITLE
[1.9.4][Crash]: ArrayIndexOutOfBoundsException

### ISSUE THREAD

**Issue by Lanse505** [EwyBoy/OreTweaker#1]
Not sure if i've derped the config or if it's a mod related crash ^^'

Crashlog:
https://gist.github.com/Lanse505/5951cc0063d84c26c449cb60ffd65331

Config:
https://gist.github.com/Lanse505/2a0e6f564a6818292848caedf8746c79

**Comment by Lanse505** [comment]
wait a second.... just noticed it references Pam's too, I'll go try disable that first before i say anything else xD

**Comment by Lanse505** [comment]
(So Tried a Second time using BoP instead of RTG Generation, that referenced Pams (See log below)
https://gist.github.com/Lanse505/00ea251590700bb0ffab592269ab7b62

disabled Pams, Booted with RTG again;
Got same crash as the first one.

**Comment by EwyBoy** [comment]
Apparently its an array out of bounds at line 33 in the WorldGenerator class. I think I found the problem. I have inserted the wrong variable into the array here.
It should be:
``int veinSize = (int) (Math.random() * (maxVeinSizes[ore] - minVeinSizes[ore]) + minVeinSizes[ore]);``
instead of: 
`int veinSize = (int) (Math.random() * (maxVeinSizes[ore] - minVeinSizes[ore]) + minVeinSizes[i]);`

I am gonna create a patch for this now. Thanks for the report.
